ironic.conf

```diff
@@ -58,6 +58,7 @@ images_path = /shared/html/tmp
 instance_master_path = /shared/html/master_images
 ipxe_enabled = true
 pxe_config_template = $pybasedir/drivers/modules/ipxe_config.template
+pxe_append_params = nofb nomodeset vga=normal console=ttyS0
```

Comment:
So conundrum is that if this is set for the physical metal, we'll end up in situations where we can't troubleshoot the physical machines but we can troubleshoot CI. Seems like we're going to need a "Is this for CI" or "non-ci" running mode?

Reply:
Oh... :-(

I didn't realize console=ttyS0 was exclusive

https://github.com/metal3-io/metal3-dev-env/pull/330 is probably useful standalone since we'd at least capture failures to PXE, I guess this part needs more thought.

Reply:
Sorry :( I feel like I'm the day darkener.

++ on logging at least PXE!

Reply:
Based on our discussion on Slack:

https://www.kernel.org/doc/html/v5.5-rc2/admin-guide/serial-console.html indicates `"You can specify multiple console= options on the kernel command line. Output will appear on all of them. The last device will be used when you open /dev/console"`